.github/workflows/ee_latest_snapshot_push.yml

```diff
@@ -70,13 +72,26 @@ jobs:
 
       - name: Build/Push EE image
         run: |
-          TAGS="--tag ${{ env.DOCKER_ORG }}/hazelcast-enterprise:${{ github.event.inputs.HZ_VERSION }}${{ matrix.suffix }}"
+          DEFAULT_JDK=11
+          if [[ "${{ matrix.jdk }}" == "$DEFAULT_JDK" ]]; then
+            TAGS="--tag ${{ env.DOCKER_ORG }}/hazelcast-enterprise:${{ github.event.inputs.HZ_VERSION }}${{ matrix.suffix }}"
+          fi
+          TAGS="${TAGS} --tag ${{ env.DOCKER_ORG }}/hazelcast:${{ github.event.inputs.HZ_VERSION }}${{ matrix.suffix }}-jdk${{ matrix.jdk }}"
           if [[ "${{ github.ref }}" == "refs/heads/master" ]]; then
-            TAGS="${TAGS} --tag ${{ env.DOCKER_ORG }}/hazelcast-enterprise:latest-snapshot${{ matrix.suffix }}"
+            if [[ "${{ matrix.jdk }}" == "$DEFAULT_JDK" ]]; then
+              TAGS="${TAGS} --tag ${{ env.DOCKER_ORG }}/hazelcast-enterprise:latest-snapshot${{ matrix.suffix }}"
+            fi
+            TAGS="${TAGS} --tag ${{ env.DOCKER_ORG }}/hazelcast-enterprise:latest-snapshot${{ matrix.suffix }}-jdk${{ matrix.jdk }}"
+          fi
+          PLATFORMS="linux/arm64,linux/amd64,linux/s390x"
+          # no JDK17 for alpine ppc64le: https://github.com/alpinelinux/aports/commit/b048defcdb41950e18262d382fab8822dc3bd8ee
+          if [[ "${{ matrix.jdk }}" != "17" ]]; then
```

Comment:
Have you checked if `ppc64le` in JDK>=17?

Reply:
it seems it's enabled again: https://github.com/alpinelinux/aports/commit/c3c3cf25a6589dad24ba220c23dba70734edd360

but I wonder should we still support all these architectures?